input/pagecontent/export.md

```diff
@@ -204,12 +204,31 @@ Export data from a FHIR server, whether or not it is associated with a patient.
       A server unable to support the requested <code>_typeFilter</code> queries SHOULD return an error and FHIR <code>OperationOutcome</code> resource so the client can re-submit a request that omits those queries. When a <code>Prefer: handling=lenient</code> header is included in the request, the server MAY process the request instead of returning an error.
       </td>
     </tr>
+    <tr>
+      <td><code>organizeOutputBy</code><br/></td>
+      <td><span class="label label-info">optional</span></td>
+      <td><span class="label label-info">optional</span></td>
+      <td>0..1</td>
+      <td><a href="https://hl7.org/fhir/valueset-resource-types.html">string of a FHIR resource type</a></td>
+      <td>When provided, a server with support for the parameter SHALL organize the resources in output files by instances of the specified resource type, including a header for each resource of the type specified in the parameter, followed by the resource and resources in the output that contain references to that resource. When omitted, servers SHALL organize each output file with resources of only single type. <a href="#bulk-data-output-file-organization">See details below</a>.<br /><br />
+      A server unable to structure output by the requested <code>organizeOutputBy</code> resource SHOULD return an error and FHIR <code>OperationOutcome</code> resource. When a <code>Prefer: handling=lenient</code> header is included in the request, the server MAY process the request instead of returning an error.
+      </td>
+    </tr>
+    <tr>
+      <td><code>allowPartialManifests</code><br/></td>
+      <td><span class="label label-info">optional</span></td>
+      <td><span class="label label-info">optional</span></td>
+      <td>0..1</td>
+      <td>boolean</td>
+      <td>When provided, a server with support for the parameter MAY return a portion of bulk data output files to a client prior to all output files being available and/or MAY distribute bulk data output files among multiple manifests and provide links for clients to page through the manifests. <a href="#manifest-link">See details below</a>.
```

Comment:
This worries me a little -- it implies the client can't know "this server provides complete pages", only "the server can give me complete pages or just keep changing a single page over time"? I'd rather see "allow partial manifests" for "the current page can change in any way at any time" and something else like "allow multiple manifest pages", with some explicit cap to say "this is the last page".

If this re-treading old design decisions, never mind me!